course/templates/course/index.html

```diff
@@ -12,85 +12,144 @@ <h1>{{ welcome_text|safe }}</h1>
 </div>
 {% endif %}
 
-<div class="row course-thumbnails">
-    {% site_advert as advert %}
-    {% if advert %}
+{% if my_instances %}
+<div class="frontpage-panel">
+    <h2 class="panel-heading">{% trans "My courses" %}</h2>
+    <div class="row course-thumbnails">
+        {% for instance in my_instances %}
```

Comment:
They are using the same template-part. I could change that.

Reply:
This has been now fixed. I left the advert to be on the front page (thus index.html) as in my mind it doesn't belong with the course-instances. And it is more an item of the front page, so it can be easily changed from there if needed.